services/src/main/java/org/keycloak/forms/login/freemarker/model/PasswordPolicyBean.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.forms.login.freemarker.model;
+
+import java.util.Set;
+
+import org.keycloak.models.RealmModel;
+
+/**
+ * @author <a href="mailto:scherer.adi@gmail.com">Adrian Scherer</a>
+ */
+public class PasswordPolicyBean {
+
+    private final RealmModel realm;
+
+    public PasswordPolicyBean(final RealmModel realmModel) {
+
+        this.realm = realmModel;
+    }
+
+    public Set<String> getPolicies() {
```

Comment:
Hello, I'm not a Java developer, but only just curious about why there is a `public Set` for me is more sense `public Get` here, sorry for bothering here, and again, only my curiosity.

Reply:
Hey, no worries, the method is a getter. The `Set` is just the returned collection that contains no duplicate elements.